zbark/example_test.go

```diff
@@ -39,3 +44,18 @@ func Example() {
 	// Output:
 	// {"level":"info","msg":"Bark accepts arbitrary types.","errors":0}
 }
+
+func ExampleDebarkify() {
+	logrusLogger := logrus.New()
+	logrusLogger.Out = os.Stdout
+	logrusLogger.Formatter = new(logrus.JSONFormatter)
+	logrusLogger.Formatter.(*logrus.JSONFormatter).TimestampFormat = "lies"
```

Comment:
can the above 2 lines be collpased into:
```go
logrusLogger.Formatter = &logrus.JSONFormatter{TimestampFormat: "..."}
```

which should also avoid the typecast